test/specs/post.js

```diff
@@ -23,14 +23,14 @@ var defaultRelease = {
   repo: 'up',
   name: 'v1.0.0',
   tag_name: 'v1.0.0',
-  target_commitish: 'bar',
+  target_commitish: 'master',
   body: 'the log'
 }
 
 test('full post run', function (t) {
   t.test('in debug mode w/o token', function (tt) {
     post({
-      options: {debug: true},
+      options: {debug: true, branch: 'master'},
```

Comment:
This PR added the usage of `options.branch` in src/post.js, and its default value is `master`.
https://github.com/semantic-release/semantic-release#options

This change just provides the default value of `options.branch` for the test code.